rskj-core/src/test/java/co/rsk/peg/PegUtilsLegacyTest.java

```diff
@@ -1793,14 +1808,72 @@ void testIsMigrationTx() {
     }
 
     @Test
-    void testIsPegOutTx() {
+    void isMigrationTx_fromP2shErpFederation_toP2shP2wshFederation() {
+        Context btcContext = new Context(networkParametersMainnet);
+        FederationContext federationContext = FederationContext.builder()
+            .withActiveFederation(p2shP2wshErpFederation)
+            .withRetiringFederation(p2shErpFederation)
```

Comment:
Shouldn't this be a `P2shP2wshErpFederation` according to the test name?

Reply:
Test says from P2shErp to P2shP2wsh. So retiring is P2shErp and active is P2shP2wsh